tests/test_model.py

```diff
@@ -29,26 +29,26 @@ def test_AstroPhot_Model(self):
 
         state = model.get_state()
 
-    def test_basic_model_methods(self):
-
-        target = make_basic_sersic()
-        model = ap.models.AstroPhot_Model(
-            name="test sersic",
-            model_type="sersic galaxy model",
-            parameters={
-                "center": [20, 20],
-                "PA": 60 * np.pi / 180,
-                "q": 0.5,
-                "n": 2,
-                "Re": 5,
-                "Ie": 1,
-            },
-            target=target,
-        )
-        rep = model.parameters.get_vector(as_representation = True)
-        nat = model.parameters.get_vector(as_representation = False)
-        self.assertTrue(torch.all(torch.isclose(rep, model.parameters.transform(nat, to_representation = True))), "transform should map between parameter natural and representation")
-        self.assertTrue(torch.all(torch.isclose(nat, model.parameters.transform(rep, to_representation = False))), "transform should map between parameter representation and natural")
+    # def test_basic_model_methods(self):
```

Comment:
Remove completely or add comment about why it should be saved and what is useful about it.

Reply:
Oh, I should add this test back in. It's a good simple test of the transofrmations